.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+    "recommendations": [
```

Comment:
It might be a good idea, to separate these for server and client.
If we want Vite plugin to work correctly we need to open client folder, then these won't be there. 
As you wish.

Reply:
Well I usually open the whole project as both client and server is written in vscode, but if you like to open them separately, I can separate them. And I think I won't use the plugin `npm run test:unit` is plenty good for me.